packages/website/pages/_document.js

```diff
@@ -24,6 +24,10 @@ class MyDocument extends Document {
             type="font/woff2"
             crossOrigin="true"
           ></link>
+          <link
+            href="https://fonts.googleapis.com/css2?family=Chivo:wght@400;700&display=swap"
+            rel="stylesheet"
+          />
```

Comment:
why do we need a new font ? and if we really do it should be processed and served from cf pages

Reply:
New font was specified by designer for blog. Further discussion led to replacing bold weight with title fonts consistent with rest of site. Will change to handle in same way as other font.